Source/Cryptlex.Net/Requests/Accounts/AccountCheckSSOEnabledData.cs

```diff
@@ -1,12 +1,14 @@
-﻿namespace Cryptlex.Net.Accounts
+using System.Text.Json.Serialization;
+namespace Cryptlex.Net.Accounts
 {
     public class AccountCheckSSOEnabledData
     {
-        public string comapnyId { get; set; }
+		[JsonPropertyName("comapnyId")]
+        public string ComapnyId { get; set; }
```

Comment:
companyId has been renamed to accountAlias on the last Cryptlex WebAPI update 

https://api.cryptlex.com/v3/docs#tag/Authentication/operation/post/v3/accounts/login/saml/verify

Reply:
Thanks! I'll get on it :)